src/safety/SafetyConfig.js

```diff
@@ -202,7 +202,8 @@ const safetyConfig = {
       //popupTemplate: null,
       showLegend:true,
       outFields:'*',
-      //baselineWhereCondition: "project = 'me8'",
+      // baselineWhereCondition: "project = 'me8'",
+      customDefaultFilter: "project = 'me8'",
```

Comment:
I've tried setting up `baselineWhereCondition`, but for some reason where clause initializes with `1=1`. `customDefaultFilter` works as intended. I need more time to figure out why this happens...